Services/CO.CDP.OrganisationInformation.Persistence/DatabaseFormRepository.cs

```diff
@@ -81,12 +90,15 @@ public async Task<IEnumerable<FormQuestion>> GetQuestionsAsync(Guid sectionId)
         return await context.Set<FormQuestion>().Where(q => q.Section.Guid == sectionId).ToListAsync();
     }
 
-    public async Task<List<FormAnswerSet>> GetFormAnswerSetsAsync(Guid sectionId, Guid organisationId)
+    public async Task<List<FormAnswerSet>> GetFormAnswerSetsFromCurrentSharedConsentAsync(Guid sectionId, Guid organisationId)
     {
-        return await context.Set<FormAnswerSet>()
-            .Include(a => a.Answers)
-            .Include(a => a.SharedConsent)
-            .Where(a => a.Section.Guid == sectionId && a.SharedConsent.Organisation.Guid == organisationId && a.Deleted == false)
+        return await context.Set<SharedConsent>()
+            .Where(x => x.Organisation.Guid == organisationId)
+            .OrderByDescending(x => x.UpdatedOn)
```

Comment:
Use `CreatedOn` in all the cases please. `UpdatedOn` won't point to the latest created shared consent if any information on shared consent is updated on one of the past records.

Reply:
done